web/packages/plugins/apps/analysis-platform/src/modules/Analysis/components/AnalysisCard.tsx

```diff
@@ -55,51 +59,71 @@ const RunAnalysisButton = (props: any) => {
 
   const analysisAbsoluteReference = `${DEFAULT_DATASOURCE_ID}/${analysis._id}`
 
-  const saveAndStartJob = () => {
+  const applicationInputHasCorrectType = (job: TJob) => {
+    if (!job.applicationInput.input.includes(job.applicationInput.inputType)) {
+      return false
+    }
+    return true
+  }
+
+  const saveAndStartJob = (task: TTask) => {
     setLoading(true)
     const runsSoFar = jobs.length
-    const job: any = {
-      name: `${analysis.task.name}-${poorMansUUID()}`,
-      triggeredBy: tokenData?.name,
-      type: JOB,
-      started: new Date(),
-      outputTarget: analysis.task.outputTarget,
-      input: analysis.task.input,
-      runner: analysis.task.runner,
-    }
-    dmssAPI.generatedDmssApi
-      .explorerAdd({
-        absoluteRef: `${analysisAbsoluteReference}.jobs`,
-        updateUncontained: false,
-        body: job,
+
+    dmssAPI
+      .getDocumentById({
+        dataSourceId: DEFAULT_DATASOURCE_ID,
+        documentId: task.applicationInput._id,
+        depth: 0,
       })
-      .then(() => {
-        // Add the new job to the state
-        // Start a job from the created job entity (last one in list)
-        jobAPI
-          .startJob(`${analysisAbsoluteReference}.jobs.${runsSoFar}`)
-          .then((result: any) => {
-            addJob(job)
-            NotificationManager.success(
-              JSON.stringify(result.data),
-              'Simulation job started'
-            )
+      .then((applicationInputEntity: TSIMAApplicationInput) => {
+        const containerJob: TJob = {
```

Comment:
```suggestion
        const job: TJob = {
```
It can be any type of job